python/xoscar/collective/tests/test_pygloo_tcp_store.py

```diff
@@ -0,0 +1,716 @@
+import multiprocessing as mp
+import os
+import platform
+import shutil
+import time
+
+import numpy as np
+
+from ...tests.core import require_linux, require_unix
+
+fileStore_path = "./collective"
+system_name = platform.system()
+
+
+def worker_allgather(rank):
+    from ..gloo import xoscar_pygloo as xp
+    from ..rendezvous import xoscar_store as xs
+
+    if rank == 0:
+        if os.path.exists(fileStore_path):
+            shutil.rmtree(fileStore_path)
+        os.makedirs(fileStore_path)
+    else:
+        time.sleep(0.5)
+
+    context = xp.rendezvous.Context(rank, 2)
+
+    if system_name == "Linux":
+        attr = xp.transport.tcp.attr("localhost")
+        dev = xp.transport.tcp.CreateDevice(attr)
+    else:
+        attr = xp.transport.uv.attr("localhost")
+        dev = xp.transport.uv.CreateDevice(attr)
+    opt = xs.TCPStoreOptions()
+    opt.port = 25001
+    opt.numWorkers = 2
+    if rank == 0:
+        opt.isServer = True
+    else:
+        opt.isServer = False
+
+    store = xs.TCPStore("127.0.0.1", opt)
+    custom_store = xp.rendezvous.CustomStore(store)
+    store = xp.rendezvous.PrefixStore(str(2), custom_store)
```

Comment:
We should wrap a `TCPStore` to `PrefixStore` directly. 
```python
store = xp.rendezvous.PrefixStore(str(2), store)
```

Reply:
Done